### PR TITLE
feat: cloud api to logs service connection for enterprise

### PR DESCRIPTION
**Pull request by exu**
Checklist:

* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/kubeshop/testkube-cloud-charts/blob/master/community/CONTRIBUTING.md).
* [x] My CI is green.

### CODE REVIEW

File: charts/testkube-enterprise/local-values.yaml [hunk 1]
```diff
@@ -223,6 +223,21 @@ testkube-cloud-api:
     apiAddress: "http://localhost:8090"
     # -- S3 bucket in which to store logs & artifacts
     outputsBucket: testkube-cloud-outputs
+    # -- External log server connection configuration
+    logServer:
```
Comment (emil2k) [inline charts/testkube-enterprise/local-values.yaml:227]: Logs service is not installed as part of the enterprise chart, it was causing issues with the certificate so it was taken out.

If you set this without the logs service will the logs be lost?

File: charts/testkube-enterprise/local-values.yaml [hunk 2]
```diff
@@ -261,6 +276,14 @@ testkube-logs-service:
     nats:
       # -- NATS URI
       uri: "nats://testkube-enterprise-nats:4222"
+    minio:
```
Comment (emil2k) [inline charts/testkube-enterprise/local-values.yaml:279]: Why was this needed? Was the installer tested again? cc @ypoplavs

Comment (exu) [reply charts/testkube-enterprise/local-values.yaml:281]: Logs service is storing data in MinIO

File: charts/testkube-enterprise/local-values.yaml (hunk 1 above)
Comment (exu) [reply charts/testkube-enterprise/local-values.yaml:227]: I've conditionally enabled it - to be able to install on local enterprise. 

disabled by default in values.

Comment (exu) [reply charts/testkube-enterprise/local-values.yaml:227]: @emil2k also renamed tls secret name

Comment (emil2k) [reply charts/testkube-enterprise/local-values.yaml:227]: @exu Thank you, we are going to try to test your changes.

File: charts/testkube-logs-service/values.yaml [hunk 3]
```diff
@@ -82,7 +82,7 @@ api:
     # -- Agent gRPCS port
     agentPort: 8443
     # -- TLS secret name which contains the certificate files TODO(emil): conflicts with agent ingress
```
Comment (emil2k) [inline charts/testkube-logs-service/values.yaml:84]: Please remove this TODO since the issue will be fixed now.

Comment (exu) [reply charts/testkube-logs-service/values.yaml:84]: done

**Review by emil2k — APPROVED** [review]
